helper/chart_data.js

```diff
@@ -118,15 +128,15 @@ exports.instanceReportChart = async function (data,reportType) {
 
          //loop the keys of matrix array
          await Promise.all(matrixRes.map(async ele => {
-            let matrixResponse = await matrixResponseObjectCreateFunc(matrixResult[ele])
+            let matrixResponse = await matrixResponseObjectCreateFunc(matrixResult[ele], type)
```

Comment:
@deepakr123 - Plan for a discussion on this with Shakthi on Monday.

Reply:
resolved